packages/appstore/src/components/routes/routes.tsx

```diff
@@ -10,9 +11,13 @@ import RouteWithSubroutes from './route-with-sub-routes.jsx';
 
 const Routes: React.FC = observer(() => {
     const { is_wallet_enabled } = useFeatureFlags();
-    const { has_wallet } = useWalletsList();
 
-    const should_show_wallets = is_wallet_enabled && has_wallet;
+    useAuthorize();
```

Comment:
we need to call `useAuthorize()` because we can't proceed with `wallet_migration` request until we cache the `authorize` response in ReactQuery cache.
we don't call `useAuthorize()` before so there is a place where i decide to call it. Maybe there is a better place to do this or better solution to call  `wallet_migration` with precached `authorize` response 🤔

Reply:
Wallet migration is a different request right ?, i dont see why we have to call authorise here 🤔